functions/serverless-functions-subscribing.adoc

```diff
@@ -0,0 +1,11 @@
+:_content-type: ASSEMBLY
+[id="serverless-functions-subscribing"]
+= Subscribing functions to CloudEvents
+include::_attributes/common-attributes.adoc[]
+:context: serverless-functions-subscribing
+
+toc::[]
+
+You can subscribe a function to a set of events, which enables you to automate responding to them.
```

Comment:
Hello, I know this description comes from the command help text, but perhaps we could re-word it to give a better description to user. But to be honest I dont have a better idea/description now.

Reply:
@jrangelramos Hi Jeff, does this look good?
`You can subscribe a function to a set of events. This links your function to `CloudEvent` objects defined by your filters and enables automated responses.`

Reply:
Yea.. sounds a little better to me!